DB2 container upgrade

Sanne opened the pull request.

Sanne committed:
Upgrade DB2 container image to version 11.5.5.0

geoand commented:
CI is complaining:

```
2021-01-08 17:51:42,451 ERROR [org.hib.eng.jdb.spi.SqlExceptionHelper] (main) [jcc][t4][2043][11550][4.27.25] Exception java.net.ConnectException: Error opening socket to server localhost/127.0.0.1 on port 50,005 with message: Connection refused (Connection refused). ERRORCODE=-4499, SQLSTATE=08001
```

famod commented:
I cannot say I'm familiar with DB2, but this might be related to the connection problems:
> [WARNING] DOCKER> Published ports are discarded when using host network mode

Sanne commented:
Interesting, thanks @famod .

I've removed the configuration changes meant for podman compatibility, let's do the DB2 upgrade first and deal with that separately.

Rebased as well.
